acquia_lift.admin.inc

```diff
@@ -256,6 +256,18 @@ function _acquia_lift_admin_build_visibility_form(&$form, &$form_state) {
 }
 
 function _acquia_lift_admin_build_advanced_confirmation_form(&$form, &$form_state) {
+  $form['acquia_lift_bootstrap_mode'] = [
+    '#type' => 'radios',
+    '#title' => t('Bootstrap mode'),
+    '#description' => t('How would you like to bootstrap Lift? Setting manul allows you to run additional scripts (useful if collecting and sending additional data or forcing an opt-in for users) but you must <a href="@docs-link" target="_blank">run the personalize() method in code</a> or Lift will not bootstrap.', array(
+      '@docs-link' => 'https://docs.acquia.com/lift/api/javascript/personalize/',
+    )),
```

Comment:
Please update description to:
```
"Automatic" means Lift scripts will automatically bootstrap and act as quickly as possible. "Manual" means Lift scripts will load but withhold itself from collecting data, delivering content, and allowing admins to login; this option is useful when you want to do things on your site (e.g. check a cookie, set field value) before you want Lift to start bootstrapping; to resume Lift's bootstrapping process, call AcquiaLiftPublicApi.personalize().
```

Reply:
Again @tottey I hope my proposed description makes sense. Please update as needed.